src/app/test/test.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { AppService } from '../app.service';
+
+
+
+@Component({
+  selector: 'app-test',
+  templateUrl: './test.component.html',
+  styleUrls: ['./test.component.scss']
+})
+export class TestComponent implements OnInit {
+  numProperties : any = [];
```

Comment:
Niente tipizzazione?

Reply:
Ho creato un'interfaccia in base al modello ritornato dall'API e ho, quindi, tipizzato il metodo.